app/javascript/orangelight/availability.es6

```diff
@@ -321,15 +324,10 @@ export default class AvailabilityUpdater {
       display_request = 'true';
     }
     // if it's on CDL then it can't be requested
-    if (availability_label_text === "Reserved for digital lending") {
+    if (cdl) {
```

Comment:
@carolyncole thank you for the question. The code never gets in the check `if (availability_label_text === "Reserved for digital lending")` because this text used to exist in the past before Alma. Now the text is 'Online' . I replaced it by checking the availability response if `cdl: true` so that it really checks if it's CDL. 
For example https://catalog-qa.princeton.edu/catalog/9976990033506421 has availability response: https://bibdata.princeton.edu/bibliographic/availability.json?deep=true&bib_ids=9976990033506421 where the cdl item is cdl: true.